Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.14
+FROM alpine:3.14.2
```

Comment:
alpine:3.14.2 comes with updated `apk-tools`, `libcrypto`, `libcurl` and `libssl` to fix the high and critical vulnerabilities:

CVE-2021-36159
CVE-2021-3711
CVE-2021-22901
CVE-2021-22945
CVE-2021-22922